Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/StoreNoOutputsToCacheTests.cs

```diff
@@ -180,6 +180,64 @@ public void RewriteProcessStoreOutputsToCacheTest()
             XAssert.IsTrue(FileExistsOnDisk(rewrittenOutput));
         }
 
+        [Fact]
+        public void RewriteCopyFileTarget()
+        {
+            const string Content = "Test";
+            var copySource = CreateSourceFile();
+            var copyTarget = CreateOutputFileArtifact();
+
+            File.WriteAllText(ArtifactToString(copySource), Content);
+            var copy = CreateAndScheduleCopyFile(copySource, copyTarget);
+
+            var rewrittenCopyTarget = copyTarget.CreateNextWrittenVersion();
+
+            var processAndOutputs = CreateAndSchedulePipBuilder(new Operation[]
+            {
+                Operation.ReadFile(copyTarget),
+                Operation.WriteFile(rewrittenCopyTarget, Content)

```

Comment:
>Content [](start = 57, length = 7)

Change the content of the file here (similarly for the rewriting process of write file below). Otherwise, your test could work if the process pip stored its outputs.

Reply:
When you specify the content in `Operation.WriteFile`, the behavior is append. So I don't need to change anything here.

---
In reply to: [277890624](https://github.com/Microsoft/BuildXL/pull/193#discussion_r277890624) [](ancestors = 277890624)